spec/lib/populate_numer_of_samples_per_pool_spec.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+# rubocop:disable RSpec/DescribeClass
+require 'rails_helper'
+require 'rake'
+
+RSpec.describe 'number_of_samples_per_pool:populate' do
+  def run_rake_task_with_args(task_name, *args)
+    Rake::Task[task_name].reenable
+    Rake.application.invoke_task("#{task_name}[#{args.join(',')}]")
+  end
+
+  shared_examples 'populating number of samples per pool' do
+    it 'populates number of samples per pool' do
+      submission = create(:submission)
+      tube = create(:tube)
+      request = create(:well_request, asset: tube, submission: submission)
+
+      # Execute
+      run_rake_task_with_args('number_of_samples_per_pool:populate', 96, submission.reload.id)
```

Comment:
Could samples_per_pool come from a let block so both run_rake_task_with_args and assertion refer to that?